PriceCalculatorsKataTests/ProductTest.cs

```diff
@@ -13,18 +14,30 @@ public class Tests
         [SetUp]
         public void Setup()
         {
-            UniversalTax.Tax = 20f;
-            UniversalDiscount.Discount = 15f;
         }
 
         [Test]
-        public void productPricePercision()
+        public void ProductPricePrecision()
         {
-            var roundedUpPriceProduct = new Product(0,20.256f)
+            UniversalTax.Tax = 20;
+            UniversalDiscount.Discount = 15;
+            UPCBasedDiscount.UPCDiscounts.TryAdd(12345, 7);
+            IPriceModifier[] taxes = {new UniversalTax()};
+            IPriceModifier[] lowPrecedenceDiscounts = {new UniversalDiscount()};
+            IPriceModifier[] highPrecedenceDiscounts = {new UPCBasedDiscount()};
+            IPriceModifier[] expenses =
+            {
+                new AbsoluteExpense(2.2f, "Transport"),
+                new RelativeExpense(1, "Packaging")
+            };
+            IPriceCalculator productPriceCalculator =
```

Comment:
this is not really a "unit testing" :P, let's see if your unit testing skills improved after taking the unit test path.